packages/dashboard-core-plugins/src/panels/IrisGridPanelTooltip.tsx

```diff
@@ -15,7 +15,10 @@ interface IrisGridPanelTooltipProps {
 function IrisGridPanelTooltip(props: IrisGridPanelTooltipProps): ReactElement {
   const { model, widgetName, glContainer, description } = props;
 
-  const rowCount = (model?.rowCount ?? 0) - (model?.pendingRowCount ?? 0);
+  const rowCount =
+    (model?.rowCount ?? 0) -
+    (model?.pendingRowCount ?? 0) -
+    (model?.floatingBottomRowCount ?? 0);
```

Comment:
You'll also need to subtract `floatingTopRowCount`. Right now the total row count is incorrect if you move aggregations to the top:
![image](https://github.com/deephaven/web-client-ui/assets/4505624/fdb9bdaf-3606-4121-9613-9165e4f4973d)